### PR TITLE
fix(Regionenkarte): prevent line search highlight from disappearing onPointerMove

### PR DESCRIPTION
**Pull request by danji90**
# How to

# Others

- [ ] It's not a hack or at least an unauthorized hack :).
- [ ] The images added are optimized.
- [ ] Everything in ticket description has been fixed.
- [ ] The author of the MR has made its own review before assigning the reviewer.
- [ ] The title means something for a human being and follows the [conventional commits](https://www.conventionalcommits.org/) specification.
- [ ] The title contains [WIP] if it's necessary.
- [ ] Labels applied. if it's a release? a hotfix?
- [ ] Tests added.

### CODE REVIEW

File: src/config/ch.sbb.regionenkarte.public/index.js [hunk 1]
```diff
@@ -65,7 +65,6 @@ export const regionenkartePublicSegment = new Layer({
       },
       properties: {
         hideInLegend: true,
-        showPopupOnHover: true,
```
Comment (danji90) [inline src/config/ch.sbb.regionenkarte.public/index.js:68]: @oterral We don't need this here anymore right? It was preventing the select highlight from lingering

Comment (danji90) [reply src/config/ch.sbb.regionenkarte.public/index.js:68]: And do we even need the whole showPopupOnHover functionality? I don't think we use it anywhere

Comment (oterral) [reply src/config/ch.sbb.regionenkarte.public/index.js:68]: If the `showPopupOnHover` property is not used anywhere, we can remove the functionnality, I guess. But check on the gitlab project too  before you do it.

**Review by oterral — APPROVED** [review]
LGTM . Nice fix. it fixes basically all the feature display when you click on a result of the search .

**Review by oterral — APPROVED** [review]

**Pull request merged**